navapbc/wic-participant-recertification-portal
# PRP-173 Create document upload bucket

## Ticket

https://wicmtdp.atlassian.net/browse/PRP-173

## Changes
* Added relevant s3 IAM perms
* Created the `doc-upload-s3` module
* Created doc upload bucket

## Context for reviewers

> The Document upload feature needs an S3 bucket. This bucket needs to be:
> 
> created in each environment
> 
> encrypted with it’s own KMS key
> 
> limited account access (for the prod bucket)

## Testing
1. Try running `terraform plan` and compare the plan to the currently deployed resources
2. Confirm resources exist in the console
3. Confirm the ability to manage the bucket from regular engineering AWS account
